metadb/scrapers/spotgen.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import spotipy
+
+
+sp = spotipy.Spotify()
+sp.trace = False
+
+def get_artist(name):
+	results = sp.search(q='artist:' + name, type='artist')
+	items = results['artists']['items']
+	if len(items) > 0:
```

Comment:
What will happen here if there are more than 1 artist with the same name? How will you be sure that you get data for the right one?
A similar situation may occur when you search for an artist and a similar named artist, but not the correct on comes up as the first result. Will you check the artist name against the `name` parameter?